README.md

```diff
@@ -4,6 +4,9 @@
 |:---------------------------|
 | This repository contains the connector and configuration code only. The implementer is responsible to acquire the connection details such as username, password, certificate, etc. You might even need to sign a contract or agreement with the supplier before implementing this connector. Please contact the client's application manager to coordinate the connector requirements.       |
 
+> :warning: **_Information_**
+> This connector requires PowerShell 7 and is designed to run in the HelloID cloudAgent. This connector is made for the PowerShell v2 tager system.
+
 <br />
```

Comment:
I have made some changed to the readme but not all. The fields are not to important to add because this will change when we the import feature will be released next month. I did however created a fields.json file which is probably not correct :innocent:

Reply:
agreed :), cannot wait for the import/export feature

Reply:
Fixed the fields.json. This can now be imported with the new feature.